packages/loaders/src/elements/Dots.tsx

```diff
@@ -61,18 +61,18 @@ export const Dots = forwardRef<SVGSVGElement, IDotsProps>(
 
     return (
       <StyledSVG
+        $dataGardenId={COMPONENT_ID}
         ref={ref}
         fontSize={size!}
         color={color!}
         width="80"
         height="72"
```

Comment:
I don't think we want these props leaking to the DOM. Also, might need to `git-blame`, but why do we have these huge `width`/`height` values?